docs/ECLWatch/ECLWa_mods/ECLWatchQueries.xml

```diff
@@ -589,33 +597,211 @@ depending on where they wind up.-->
+          role="bold">Package Parts</emphasis> tab. Then press the <emphasis
+          role="bold">Remove</emphasis> button and confirm that you really
+          want to remove the part.</para>
+
+          <para><emphasis role="bold">Get Part </emphasis><figure>
+              <title>Get Part</title>
+
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="../../images/ECLWaGetPart01.jpg"
+                             vendor="ECLWatchSS" />
+                </imageobject>
+              </mediaobject>
+            </figure></para>
+
+          <para>Press the <emphasis role="bold">Get Part</emphasis> button to
+          view the contents of the selected part. <figure>
+              <title>Package Part Contents</title>
+
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="../../images/ECLWaGetPart02.jpg"
+                             vendor="ECLWatchSS" />
+                </imageobject>
+              </mediaobject>
+            </figure></para>
+        </sect3>
+
+        <sect3>
+          <title id="Packagemap_Validation">Package Map Validation</title>
+
+          <para>To validate your package map:</para>
+
+          <para><figure>
+              <title>Validate Package Maps</title>
+
+              <mediaobject>
+                <imageobject>
+                  <imagedata fileref="../../images/ECLWaPkMgr.jpg" />
+                </imageobject>
+              </mediaobject>
+            </figure></para>
 
-        <para>Choose the <emphasis role="bold">Target</emphasis> and <emphasis
-        role="bold">Process</emphasis> from the drop lists on the Validate
-        Package Map tab.</para>
+          <orderedlist>
+            <listitem>
+              <para>Select the <emphasis role="bold">Validate Package
+              Map</emphasis> tab</para>
+            </listitem>
 
-        <para>Press the <emphasis role="bold">Validate</emphasis> button to
-        validate the package map. The result is shown on the <emphasis
-        role="bold">Validate Active Package Map</emphasis> tab. You can also
-        Validate the package content, on the V<emphasis role="bold">alidate
-        Package Content</emphasis> tab.</para>
+            <listitem>
+              <para>Choose the <emphasis role="bold">Target</emphasis> and
+              <emphasis role="bold">Process</emphasis> from the drop lists on
+              the <emphasis role="bold">Validate Package Map</emphasis>
+              tab.</para>
+            </listitem>
+
+            <listitem>
+              <para>Press the <emphasis role="bold">Validate</emphasis> button
+              to validate the package map.</para>
+            </listitem>
+          </orderedlist>
+
+          <para>The result is shown on the <emphasis role="bold">Validate
+          Active Package Map</emphasis> tab.</para>
+
+          <para>You can also Validate the package content on the <emphasis
```

Comment:
It's not really "the" content.  The Validate package content allows you to open any package file, or insert any package content into the form and validate it.  It does not first have to be published onto the system.